portal/date_tools.py

```diff
@@ -137,5 +137,5 @@ def localize_datetime(dt, user):
         best = local.astimezone(tz)
     else:
         best = dt
-    d, m, y = best.strftime('%-d %b %Y').split()
+    d, m, y = best.strftime('%d %b %Y').split()
```

Comment:
Was this intentional? I think this will remove any leading `0` on the day of the month:
```
>>> f
datetime.datetime(2018, 6, 9, 16, 28, 1, 964588)
>>> f.strftime('%d %b %Y')
'09 Jun 2018'
>>> f.strftime('%-d %b %Y')
'9 Jun 2018'
```

Reply:
Running it with "%-d', I'm getting a ValueError, but with '%d' it's not removing any leading 0.

Reply:
It looks like this is [a quirk of windows](https://stackoverflow.com/questions/28894172/why-does-d-or-e-remove-the-leading-space-or-zero/28916983#28916983). Please revert 
>Python datetime.strftime() delegates to C strftime() function that is platform-dependent:

[See also](https://github.com/facebook/prophet/issues/99)

[Possible solution here](https://stackoverflow.com/questions/904928/python-strftime-date-without-leading-0/16097385#16097385)